site/blog/unlisted/deploy-ghost-ec2.mdx

```diff
@@ -146,7 +152,7 @@ SELECT
 SELECT
   *
 FROM
-  iasql_commit();
+  iasql_commit ();
```

Comment:
does that still work?

Reply:
Yes, though I don't know why you would format that way.

Reply:
it's just prettier formatting it